packages/docs/src/shared-components/IconSearchPageThemes.tsx

```diff
@@ -52,6 +61,56 @@ export const IconSearchPageThemes: FC<Props> = ({ themes }) => {
               </syn-tab-panel>
             </Fragment>
           ))}
+
+          <syn-tab panel="migration" slot="nav" active>
+            Icon Migration
+          </syn-tab>
+
+          <syn-tab-panel name="migration">
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                flexWrap: 'wrap',
+                gap: 'var(--syn-spacing-large)',
+              }}
+            >
+              {Object.entries(sick2018Icons).map(([iconName]) => (
+                <div
+                  style={{
+                    border: '1px solid var(--syn-color-neutral-200)',
+                    borderRadius: 'var(--syn-border-radius-medium)',
+                    padding: 'var(--syn-spacing-large)',
+                    textAlign: 'center',
+                    width: '150px',
+                  }}
+                  key={iconName}
+                >
+                  <h2
+                    style={{
+                      textWrap: 'pretty',
+                      wordBreak: 'break-word',
+                      fontSize: 'var(--syn-font-size-small)',
+                      marginBottom: 'var(--syn-spacing-medium)',
+                    }}
+                  >{iconName}</h2>
+                  <div
+                    style={{
+                      display: 'flex',
+                      flexDirection: 'row',
+                      fontSize: '30px',
+                      gap: 10,
+                      justifyContent: 'center',
+                    }}
+                  >
+                    <syn-icon name={iconName} library="default" />
+                    <syn-icon name={iconName} library="migration" />
```

Comment:
i am not sure which icon set it is using here for the "default". I guess the one you selected in the mode switch? I think this should always be the 2018 to compare them or even the 2025 to see which we migrated?

I think we should register an explicit library here like I did in the `IconSearchPage.tsx` or just reuse it from there and maybe show all 3 icon libraries?
Like: 

```suggestion
                    <syn-icon name={iconName} library="sick2018" />
                    <syn-icon name={iconName} library="sick2025-outline" />
                    <syn-icon name={iconName} library="migration" />
```